compiler/config/tests.config

```diff
@@ -15,20 +15,30 @@ bin    = ./scripts/check-cct
 okdirs = !CCT/success
 kodirs = !CCT/fail
 
-[test-all]
-okdirs = !examples !tests/success
-kodirs = tests/fail tests/fail/annotation
-exclude = examples/arm_m4
+[test-x86-64-ATT]
+bin     = ./scripts/check-x86-64
+args    = -ATT
+okdirs  = examples/**/x86-64 tests/success/**/x86-64
+kodirs  = tests/fail/**/x86-64
+exclude = tests/fail/warning/x86-64
 
-[test-print]
+[test-x86-64-Intel]
+bin = ./scripts/check-x86-64
+args = -intel
+okdirs = examples/**/x86-64 tests/success/**/x86-64
+kodirs = tests/fail/**/x86-64
+exclude = tests/fail/warning/x86-64
+
+[test-x86-64-print]
 bin = ./scripts/parse-print-parse
-okdirs = tests/success
+okdirs = tests/success/x86-64
 
-[test-extraction]
```

Comment:
Please change the CI job referring to this name (`.gitlab-ci.yml`, line 249).

Reply:
Done.